src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
```

Comment:
Ensure that the `getUser` function is correctly imported from the `../../api` file. Verify that the `../../api` file exists and exports the `getUser` function as expected.